csharp/src/Google.Protobuf/Reflection/ExtensionCollection.cs

```diff
@@ -107,13 +107,11 @@ internal void CrossLink()
             {
                 descriptor.CrossLink();
 
-                IList<FieldDescriptor> _;
-                if (!declarationOrder.TryGetValue(descriptor.ExtendeeType, out _))
-                {
-                    declarationOrder.Add(descriptor.ExtendeeType, new List<FieldDescriptor>());
-                }
+                IList<FieldDescriptor> list;
+                if (!declarationOrder.TryGetValue(descriptor.ExtendeeType, out list))
```

Comment:
- in protobuf codebase, always use curly brackets for `if` statements (single statements ifs and loops are considered dangerous).
- I think 
```
list = new List<FieldDescriptor>();
declarationOrder.Add(......., list);
```
is more readable.
```

Reply:
Could we add an editorconfig with `csharp_prefer_braces` enabled to prevent style issues like this and others in the future?

Reply:
At some point, I'd like to add a set of rules to enforce code style in the protobuf C# codebase. I'm not sure what the best approach is (there are multiple ways of doing this).